odata-url-conventions/4 Resource Path.md

```diff
@@ -952,8 +952,9 @@ For content-type `text/plain`, it MUST use the same percent-encoding as in
 URLs (especially: no spaces, tabs, or line breaks allowed) and MUST
 follow the syntax rules described in chapter Query Options.
 For content-type `application/x-www-form-urlencoded`, it MUST first percent-decode
-the request body and then replace `+` with `%20`. The result MUST then be
-treated as the query portion of the URL like for content-type `text/plain`.
+the request body except for `%26` (ampersand) and `%3D` (equals) and then
+replace `+` with `%20`. The result MUST then be treated as the query portion
```

Comment:
Actually: decode `+` as space. And we have https://issues.oasis-open.org/browse/ODATA-1540 for allowing `+` as an encoding for space, so servers supporting that don't need special handling here.